test

```diff
@@ -149,18 +149,22 @@ function fmt_tests {
 
 	echo "Checking commit titles..."
 	git log master..HEAD --oneline | while read l; do
-		pfx=`echo "$l" | cut -f2 -d' '`
-		if [ "$pfx" == "Merge" ]; then
+		commitMsg=`echo "$l" | cut -f2- -d' '`
+		if [[ "$commitMsg" =~ Merge* ]]; then
```

Comment:
`==` instead of `=~` -- the `=~` will match any substring, `==` matches just the beginning